<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonScan">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>start scan</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_output">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">Scan resule</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_waring">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>please make sure that you can reach the device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExist">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>420</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>exist</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IP">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>140</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lable_IPtarget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP target:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_picture">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="picture.qrc">:/new/prefix1/C:/Users/weiyu/Pictures/github.gif</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMore_select">
    <property name="title">
     <string>More selects</string>
    </property>
    <addaction name="actionScan_paremeters"/>
   </widget>
   <addaction name="menuMore_select"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScan_paremeters">
   <property name="text">
    <string>Scan parameters</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonExist</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
